### PR TITLE
730.hwi.connection pool queue error

### PR DESCRIPTION
**Pull request by gbasolu**

### CODE REVIEW

File: hydra_base/db/__init__.py [hunk 1]
```diff
@@ -120,18 +127,32 @@ def connect(db_url=None):
 
     global engine
 
-    db_pool_size = config.get('mysqld', 'pool_size', 10)
-    db_pool_recycle = config.get('mysqld', 'pool_recycle', 300)
-    db_max_overflow = config.get('mysqld', 'max_overflow', 10)
+    # Lets use at least 10 for size and 20 for overflow (hydra.ini file)
+    # To test the timeout: pool_size:1, max_overflow: 0, pool_timeout: 5 or any low value
+    db_pool_size = int(config.get('mysqld', 'pool_size',1)) # 10
```
Comment (knoxsp) [inline hydra_base/db/__init__.py:132]: This should be 10

File: hydra_base/db/__init__.py [hunk 2]
```diff
@@ -120,18 +127,32 @@ def connect(db_url=None):
...
+    db_pool_recycle = int(config.get('mysqld', 'pool_recycle', 1)) # 300
```
Comment (knoxsp) [inline hydra_base/db/__init__.py:133]: Should this be 1?

File: hydra_base/db/__init__.py [hunk 3]
```diff
@@ -120,18 +127,32 @@ def connect(db_url=None):
...
+    db_max_overflow = int(config.get('mysqld', 'max_overflow', 2)) # 10 -> 30
```
Comment (knoxsp) [inline hydra_base/db/__init__.py:134]: This should be 20 or 30

File: hydra_base/db/__init__.py [hunk 4]
```diff
@@ -110,7 +117,7 @@ def connect(db_url=None):
     if db_url is None:
         db_url = config.get('mysqld', 'url')
 
-    log.info("Connecting to database")
+    log.info("Connecting to database 1.0")
```
Comment (knoxsp) [inline hydra_base/db/__init__.py:120]: why 1.0?

File: hydra_base/db/__init__.py [hunk 5]
```diff
@@ -120,18 +127,32 @@ def connect(db_url=None):
...
-    db_pool_recycle = config.get('mysqld', 'pool_recycle', 300)
-    db_max_overflow = config.get('mysqld', 'max_overflow', 10)
+    # Lets use at least 10 for size and 20 for overflow (hydra.ini file)
+    # To test the timeout: pool_size:1, max_overflow: 0, pool_timeout: 5 or any low value
+    db_pool_size = int(config.get('mysqld', 'pool_size',1)) # 10
+    db_pool_recycle = int(config.get('mysqld', 'pool_recycle', 1)) # 300
+    db_max_overflow = int(config.get('mysqld', 'max_overflow', 2)) # 10 -> 30
+    db_pool_timeout = int(config.get('mysqld', 'pool_timeout', 10))
+
+    log.warning(f"db_pool_size: {db_pool_size}")
```
Comment (knoxsp) [inline hydra_base/db/__init__.py:137]: Please put these into 1 line instead of multiple

File: hydra_base/db/__init__.py (hunk 4 above)
Comment (gbasolu) [reply hydra_base/db/__init__.py]: Just a check

File: hydra_base/db/__init__.py (hunk 1 above)
Comment (gbasolu) [reply hydra_base/db/__init__.py]: Done

**Pull request merged**